templates/cli/src/Client.php.twig

```diff
@@ -0,0 +1,324 @@
+<?php
+
+namespace {{ spec.title | caseUcfirst }};
+
+use Exception;
+use Utopia\CLI\Console;
+
+class Client
+{
+    const METHOD_GET = 'GET';
+    const METHOD_POST = 'POST';
+    const METHOD_PUT = 'PUT';
+    const METHOD_PATCH = 'PATCH';
+    const METHOD_DELETE = 'DELETE';
+    const METHOD_HEAD = 'HEAD';
+    const METHOD_OPTIONS = 'OPTIONS';
+    const METHOD_CONNECT = 'CONNECT';
+    const METHOD_TRACE = 'TRACE';
+
+    const USER_PREFERENCES_FILE = __DIR__."/.prefs.json";
+    const PREFERENCE_ENDPOINT = "endpoint";
```

Comment:
Is this a file path?

Reply:
No it is a key for a key => value pair in the preferences array. (Similar to X-Appwrite-Project etc.)